src/task/task4/ListOfNumbers.java

```diff
@@ -0,0 +1,40 @@
+package task.task4;
+
+import java.util.ArrayList;
+
+import static java.lang.System.*;
+
+public class ListOfNumbers {
+    ArrayList<Double> skaiciusSarasas;
+
+    public ListOfNumbers() {
+        this.skaiciusSarasas = new ArrayList<>();
+    }
+
+    public void addNumber(Double num) {
+        this.skaiciusSarasas.add(num);
+    }
+
+    public Double getAverage() {
+        Double sum = 0d;
+        Double average = null;
+        for (Double d : skaiciusSarasas) {
+            sum += d;
+        }
+        try {
+            average = sum / skaiciusSarasas.size();
+
+        } catch (Exception e) {
+            out.println(e.getMessage());
+        }
```

Comment:
Nerekomenduočiau daryti try catch tik dėl to, kad sugaudyti atveji kai listas tuščias. Rekomenduočiau tiesiog padaryti paprastą patikrinimą funkcijos pradžioje `if(size == null){ return 0.0; ]`

Reply:
o ne geriau tada rašyt  **return null**? Nes 0.0 gali būt realus vidurkis.

Reply:
galima, čia jau i taip i taip tinka, nuo biznio reikalavimų priklauso:)